process/slash/detector/common_test.go

```diff
@@ -0,0 +1,262 @@
+package detector_test
+
+import (
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/ElrondNetwork/elrond-go-core/data"
+	"github.com/ElrondNetwork/elrond-go-core/data/endProcess"
+	coreSlash "github.com/ElrondNetwork/elrond-go-core/data/slash"
+	"github.com/ElrondNetwork/elrond-go-core/hashing"
+	"github.com/ElrondNetwork/elrond-go-core/marshal"
+	crypto "github.com/ElrondNetwork/elrond-go-crypto"
+	llsig "github.com/ElrondNetwork/elrond-go-crypto/signing/mcl/multisig"
+	"github.com/ElrondNetwork/elrond-go-crypto/signing/mcl/singlesig"
+	"github.com/ElrondNetwork/elrond-go-crypto/signing/multisig"
+	mockEpochStart "github.com/ElrondNetwork/elrond-go/epochStart/mock"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/process/headerCheck"
+	"github.com/ElrondNetwork/elrond-go/process/mock"
+	"github.com/ElrondNetwork/elrond-go/process/slash"
+	"github.com/ElrondNetwork/elrond-go/process/slash/detector"
+	"github.com/ElrondNetwork/elrond-go/sharding"
+	shardingMock "github.com/ElrondNetwork/elrond-go/sharding/mock"
```

Comment:
can you import mocks from testscommon instead?

Reply:
Replaced only `&shardingMock.NodesCoordinatorCacheMock{}`. Other mocks were not in testscommon and I think it's better not to extend this PR with any further refactoring

Reply:
All new code should import from testscommon, otherwise we add more refactor work for later.

Reply:
Refactored them